### PR TITLE
Update Security policy

### PR DESCRIPTION
**Pull request by omalyshe**
### Description 
Security policy updated to include vulnerability reporting reference.

Fixes # - _issue number(s) if exists_

- [x] - git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/oneapi-src/oneTBB/blob/master/CONTRIBUTING.md#pull-requests) for details)_

### Type of change

_Choose one or multiple, leave empty if none of the other choices apply_

_Add a respective label(s) to PR if you have permissions_

- [ ] bug fix - _change that fixes an issue_
- [ ] new feature - _change that adds functionality_
- [ ] tests - _change in tests_
- [ ] infrastructure - _change in infrastructure and CI_
- [x] documentation - _documentation update_

### Tests

- [ ] added - _required for new features and some bug fixes_
- [x] not needed

### Documentation

- [ ] updated in # - _add PR number_
- [ ] needs to be updated
- [x] not needed

### Breaks backward compatibility
- [ ] Yes
- [x] No
- [ ] Unknown

### Notify the following users
_List users with `@` to send notifications_

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -1,7 +1,66 @@
...
+As an open-source project, we understand the importance of and responsibility
+for security. This Security policy outlines our guidelines and procedures for
+ensuring the highest level of Security and trust for our users who consume
+oneTBB.
 
-## Reporting a Vulnerability
-Please report any security vulnerabilities in this project 
-[utilizing the guidelines here](https://www.intel.com/content/www/us/en/security-center/vulnerability-handling-guidelines.html).
+## Supported Versions
+Security vulnerabilities can be fixed in the [latest version][1] version
```
Comment (rozhukov) [inline SECURITY.md:8]: Minor suggestion: 
Security vulnerabilities are fixed in the [latest version][1] version...

**Review by rozhukov — CHANGES_REQUESTED** [review]
I left I minor change proposal. Rather than that - looks good to me.
Thank you team!

File: SECURITY.md (hunk 1 above)
Comment (omalyshe) [reply SECURITY.md]: Changes applied. Thank you.

**Review by pavelkumbrasev — APPROVED** [review]
LGTM!

File: SECURITY.md [hunk 2]
```diff
@@ -1,7 +1,66 @@
...
+
+## Report a Vulnerability
+We are very grateful to the security researchers and users that report back
+security vulnerabilities. We investigate every report thoroughly.
+We strongly encourage you to report security vulnerabilities to us privately,
+before disclosing them on public forums or opening a public GitHub issue. 
+Report a vulnerability to us in one of two ways:
+* Open a draft [**GitHub Security Advisory**][2]
+* Send e-mail to the following address: **security@lists.uxlfoundation.org**.
+Along with the report, please include the following info:
```
Comment (aepanchi) [inline SECURITY.md:20]:
```suggestion
Along with the report, provide the following info:
```

File: SECURITY.md [hunk 3]
```diff
@@ -1,7 +1,66 @@
...
+  * A descriptive title.
+  * Your name and affiliation (if any).
+  * A description of the technical details of the vulnerabilities.
+  * A minimal example of the vulnerability so we can reproduce your findings.
+  * An explanation of who can exploit this vulnerability, and what they gain
+  when doing so.
```
Comment (aepanchi) [inline SECURITY.md:26]:
```suggestion
  doing so. 
```

File: SECURITY.md [hunk 4]
```diff
@@ -1,7 +1,66 @@
...
+* Send e-mail to the following address: **security@lists.uxlfoundation.org**.
+Along with the report, please include the following info:
+  * A descriptive title.
+  * Your name and affiliation (if any).
+  * A description of the technical details of the vulnerabilities.
+  * A minimal example of the vulnerability so we can reproduce your findings.
+  * An explanation of who can exploit this vulnerability, and what they gain
+  when doing so. 
+  * Whether this vulnerability is public or known to third parties. If it is,
+  please provide details.
```
Comment (aepanchi) [inline SECURITY.md:28]:
```suggestion
  provide details.
```

File: SECURITY.md [hunk 5]
```diff
@@ -1,7 +1,66 @@
...
+component, try to report directly there first.
+
+### When Should I NOT Report a Vulnerability?
+* You got an automated scan hit and are unable to provide details.
+* You need help using oneTBB for security.
+* You need help applying security-related updates.
+* Your issue is not security-related.
+
+## Security Reports Review Process
+Our goal is to respond quickly to your inquiry, and to coordinate a fix and
```
Comment (aepanchi) [inline SECURITY.md:44]:
```suggestion
We aim to respond quickly to your inquiry and coordinate a fix and
```

File: SECURITY.md [hunk 6]
```diff
@@ -1,7 +1,66 @@
...
+are fixed in the next planned release.
+
+## Disclosure Policy
+We will publish security advisories using the 
+[**GitHub Security Advisories feature**][3]
+to keep our community well-informed, and will credit you for your findings
+unless you prefer to stay anonymous. We request that you refrain from
+exploiting the vulnerability or making it public before the official disclosure.
+
+We will disclose the vulnerabilities and/or bugs as soon as possible once
```
Comment (aepanchi) [inline SECURITY.md:56]:
```suggestion
We will disclose the vulnerabilities and bugs as soon as possible once
```

File: SECURITY.md [hunk 7]
```diff
@@ -1,7 +1,66 @@
...
+mitigation is implemented and available. 
+
+## Feedback on This Policy
+If you have any suggestions on how this Policy could be improved, please submit
```
Comment (aepanchi) [inline SECURITY.md:60]:
```suggestion
If you have any suggestions on how this Policy could be improved, submit
```

File: SECURITY.md [hunk 8]
```diff
@@ -1,7 +1,66 @@
...
+an issue or a pull request to this repository. Please **do not** report
```
Comment (aepanchi) [inline SECURITY.md:61]:
```suggestion
an issue or a pull request to this repository. **Do not** report
```

**Review by aepanchi — APPROVED** [review]
LGTM

**Review by rozhukov — APPROVED** [review]

**Pull request merged**